### ISSUE TITLE
problems authenticating for Stadia Maps

### ISSUE THREAD

**Issue by jdmarshl** [berkeley-dsep-infra/hubs#5503]
### Bug description

I am getting a 401 authentication when I try to use Stadia Maps tiles in Datahub https://docs.stadiamaps.com/authentication/

### Environment & setup

- Datahub
- Python

I should note that this problem does not occur when I run the same notebook locally using Anaconda, and it looks like Datahub is behaving differently with the tilesets in Folium than it did last year

### How to reproduce

sfmap_stamen_toner = folium.Map(sf_coords, 
                                tiles = 'https://tiles.stadiamaps.com/tiles/stamen_toner/{z}/{x}/{y}{r}.png', 
                                attr='&copy; <a href="https://www.stadiamaps.com/" target="_blank">Stadia Maps</a> &copy; <a href="https://www.stamen.com/" target="_blank">Stamen Design</a> &copy; <a href="https://openmaptiles.org/" target="_blank">OpenMapTiles</a> &copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors',
                                zoom_start = 12)
sfmap_stamen_toner

**Comment by shaneknapp** [comment]
i don't think it's datahub, i believe it's the move from stam to stadiamaps....  i believe you need to register and create an API key as per https://docs.stadiamaps.com/guides/migrating-from-stamen-map-tiles

https://docs.stadiamaps.com/guides/migrating-from-stamen-map-tiles#folium

fwiw, we haven't changed anything on datahub WRT folium et al.  i will try and look a little deeper in to this on monday.

**Comment by jdmarshl** [comment]
Thank you. I noticed that heatmap in Folium seems to sometimes stop
displaying the overlay. I cannot figure out why.

I did, however, register with Stadia Maps and now it seems to work on
Datahub; it does not require that when you run it locally, says the
documentation.

Still, though, heatmap was behaving in an unstable way in terms of
displaying the overlay. Thank you for your help.

Jon Marshall, Ph.D. (he/him)
Assistant Dean
JSP/Legal Studies
